cypress/support/step-definitions/flat-table-steps.js

```diff
@@ -210,3 +211,39 @@ Then("The whole row is highlighted", () => {
 Then("Flat table caption is set to {word}", (text) => {
   flatTableCaption().should("have.text", text);
 });
+
+When("I click on the {word} cell in the first row", (position) => {
+  flatTableCell(positionOfElement(position)).click();
+});
+
+Then("The subrows are visible", () => {
+  flatTableSubRow().should("be.visible");
+});
+
+Then("The subrows are not visible", () => {
+  flatTableSubRow().should("not.exist");
+});
+
+When("I click on the first cell", () => {
+  flatTableCell(0).click();
+});
+
+Then("The sub rows have the correct darker background colour", () => {
+  flatTableSubRow()
+    .children()
+    .should("have.css", "background-color", "rgb(250, 251, 251)");
```

Comment:
this colour `rgb(250, 251, 251)` is not so dark :), rather almost white

Reply:
It was just because it was darker than the parent rows, but agree it is not dark